### DISCUSSION TITLE
Clarification of `networkx.karate_club_graph()` Dataset

### DISCUSSION THREAD

**Discussion by ivanov** [networkx/networkx#7515] [Q&A]
I received this report by email, circling back with the author to see if he has a github account

> Subject: Potential Bug in `networkx.karate_club_graph()` Dataset
> Dear Paul Ivanov,
> 
> I am not sure if you are the correct contact for this issue, but I hope you
> can direct this to the appropriate NetworkX developers.
> 
> I am writing because I have identified a potential mistake in the dataset
> used in networkx.karate_club_graph(). Recently, while working on a project
> involving Zachary's Karate Club, I noticed the following discrepancy:
> 
> In networkx.karate_club_graph(), network.nodes[8]['club'] contains 'Mr. Hi'
> instead of 'Officer'. According to the publication "An Information Flow
> Model for Conflict and Fission in Small Groups" (see Table 3), this node
> should belong to 'Officer'.
> 
> I hope this information is helpful and that any necessary corrections can
> be made.

From what I can tell, the "club" entries in the karate club data uses "Club After Fission" (Column 4) of Table 1, which is repeated as "Club After Split From Data" (column 5) in Table 3 of the original paper, which does indeed have "Mr. Hi's" at the 9th row (accommodating for the change of index from 1-based to 0-based), so I don't think there's a bug here. In fact, in comments on #5820, both @dschult and @rossbar confirm this. 

Does this confusion come up often? Would it be worth specifically stating in the documentation or as a comment that the karate data comes from Figure 3 of the original paper coupled with "Club After Fission" column of Table 1.

**Comment by rossbar** [comment]
I agree @ivanov - given that it's come up at least twice, it's probably worth a note in the docstring!
